packages/web-frontend/src/components/View/ChartWrapper/parseChartConfig.js

```diff
@@ -73,9 +73,9 @@ const getDefaultPaletteName = (chartType, numberRequired) => {
 const sortChartConfigByLegendOrder = chartConfig => {
   return Object.entries(chartConfig)
     .sort(([, cfg1], [, cfg2]) => {
-      if (Number.isNaN(cfg1.legendOrder) && Number.isNaN(cfg2.legendOrder)) return 0;
-      if (Number.isNaN(cfg1.legendOrder)) return -1;
-      if (Number.isNaN(cfg2.legendOrder)) return 1;
+      if (isNaN(cfg1.legendOrder) && isNaN(cfg2.legendOrder)) return 0;
+      if (isNaN(cfg1.legendOrder)) return 1;
+      if (isNaN(cfg2.legendOrder)) return -1;
```

Comment:
Number.isNaN usually check if value is equal to 'NaN', it will return false for any other value. e.g.

isNaN(NaN) //true
Number.isNaN(NaN) //true

isNaN('abc') //true
Number.isNaN('abc') //false

isNaN(123) //false
Number.isNaN(123) //false

isNaN(undefined) //true
Number.isNaN(undefined) //false